test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -762,6 +764,24 @@ void ProtoConverter::visit(NullaryOp const& _x)
 			m_output << dictionaryToken();
 		break;
 	}
+	case NullaryOp::BASEFEE:
+		// Replace calls to basefee() on unsupported EVMs with a dictionary
+		// token.
+		if (m_evmVersion.hasBaseFee())
+			m_output << "basefee()";
+		else
+			m_output << dictionaryToken();
+		break;
+	}
```

Comment:
Actually, syntax is broken here. This does not build because of the extra `}`.

Reply:
Sorry, fixed this now.